easybuild/easyconfigs/a/adjustText/adjustText-0.7.3-intel-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'adjustText'
+version = '0.7.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/Phlya/adjustText'
+description = "A small library for automatically adjustment of text position in matplotlib plots to minimize overlaps."
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+toolchainopts = {'usempi': True}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['b90e275a95b4d980cbbac7967914b8d66477c09bc346a0b3c9e2125bba664b06']
+
+download_dep_fail = True
+use_pip = True
+
+dependencies = [
```

Comment:
@boegel maybe move this to above `download_dep_fail`?
also, don't you have to add `check_ldshared = True`?

Reply:
fixed in e61e30ddc0

`check_ldshared` is not required here, since nothing is being compiled here, this is a pure Python package (in fact, there's basically only `adjustText/__init__.p`, all the code is shoved in there...)